rust-version

```diff
@@ -1 +1 @@
-538f118da1409759ba198acc0ff62070bc6d2dce
+a24a020e6d926dffe6b472fc647978f92269504e
```

Comment:
If you only bump to edabf59ca4646b3fc1a961c26431215001043f6a, does that help for CI?

Reply:
That worked, thanks.